packages/fx-core/src/component/generator/copilotPlugin/helper.ts

```diff
@@ -291,6 +291,13 @@ function sortOperations(operations: ListAPIInfo[]): ApiOperation[] {
       id: operation.api,
       label: operation.api,
       groupName: arr[0],
+      detail: !operation.auth
+        ? getLocalizedString("core.copilotPlugin.api.noAuth")
+        : Utils.isBearerTokenAuth(operation.auth.authScheme)
+        ? getLocalizedString("core.copilotPlugin.api.apiKeyAuth")
+        : Utils.isOAuthWithAuthCodeFlow(operation.auth.authScheme)
+        ? "OAuth"
```

Comment:
maybe "OAuth" should be also put into package.nls.json file?

Reply:
Does it need any localization?

Reply:
Looking at this https://learn.microsoft.com/zh-cn/entra/identity-platform/v2-oauth2-auth-code-flow OAuth is not localized.